src/java/fr/paris/lutece/portal/service/security/LuteceUser.java

```diff
@@ -352,11 +353,8 @@ public void setGroups( Collection<String> groups )
         else
         {
             newArray = new String [ collection.size( ) + array.length];
-
-            for ( j = 0; j < array.length; j++ )
-            {
-                newArray [j] = array [j];
-            }
+            
+            newArray = Arrays.copyOf(array, collection.size( ) + array.length );

```

Comment:
is the previous `newArray` instantiation still necessary ?

Reply:
Corrected in another commit of the same PR.